server/src/main/java/org/apache/iotdb/db/query/externalsort/serialize/TimeValuePairDeserializer.java

```diff
@@ -0,0 +1,33 @@
+/**
+ * Copyright © 2019 Apache IoTDB(incubating) (dev@iotdb.apache.org)
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.iotdb.db.query.externalsort.serialize;
+
+import java.io.IOException;
+import org.apache.iotdb.db.query.reader.IPointReader;
+import org.apache.iotdb.db.query.reader.IReaderByTimestamp;
+import org.apache.iotdb.db.utils.TimeValuePair;
+
+public interface TimeValuePairDeserializer extends IPointReader, IReaderByTimestamp {
```

Comment:
maybe IExternalFileDeserializer?

Reply:
Not quite satisfying too, since the main point may not be about "FileDeserializer", actually, I can hardly see anything about "deserialize" from its interfaces.
But it is fine, we can put this aside.